app/models/profile.rb

```diff
@@ -27,6 +27,14 @@ class Profile < ApplicationRecord
 
   attr_reader :personal_key
 
+  def fraud_review_pending?
+    !!(fraud_review_pending || fraud_review_pending_at)
```

Comment:
Presuming that the || and !! business is interim code, until the booleans come out

Reply:
The `||` is temporary. The `!!` converts nil, timestamps, etc. to true/false, which will continue to be useful for a query method.